src/guc.h

```diff
@@ -29,6 +31,21 @@ extern TSDLLEXPORT char *ts_guc_license_key;
 extern char *ts_last_tune_time;
 extern char *ts_last_tune_version;
 extern char *ts_telemetry_cloud;
+extern TSDLLEXPORT bool ts_guc_enable_2pc;
```

Comment:
Is there a good reason to keep this tweak beside testing?

Reply:
Maybe not. Still, 1pc remains an option for potentially faster inserts at the risk of inconsistencies in case of failures. Unknown what the effect on performance is though.